contracts/test/integration/IntegrationTestMaplePCVDeposit.t.sol

```diff
@@ -0,0 +1,643 @@
+
+        // Someone creates a loan
+        address mapleLoanFactory = 0x36a7350309B2Eb30F3B908aB0154851B5ED81db0;
+        address mapleDebtLockerFactory = 0xA83404CAA79989FfF1d84bA883a1b8187397866C;
+        address mapleLoan;
+        uint256 gracePeriod = 432000; // 5 days
+        uint256 paymentInterval = 2592000; // 30 days
+        bytes memory arguments = abi.encode(
+            address(this), // borrower
+            address(0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599), // assets[0] = collateral = WBTC
+            MainnetAddresses.USDC, // assets[1] = principal = USDC
+            gracePeriod, // terms[0]
+            paymentInterval, // terms[1] = paymentInterval
+            2, // terms[2] = numberOfPayments
+            0, // amounts[0] = collateralRequired
+            targetUsdcBalance, // amounts[1] = principalRequested
+            targetUsdcBalance, // amounts[2] = endingPrincipal
+            0.12e18, // rates[0] = interestRate = 12%
+            0.02e18, // rates[1] = earlyFeeRate = 2%
+            0, // rates[2] = lateFeeRate = 0
+            0.05e18 // rates[3] = lateInterestPremium = 5%
+        );
+        mapleLoan = IMapleLoanFactory(mapleLoanFactory).createInstance(
+            arguments,
+            keccak256(
+                bytes("some salt for testAccrueLossesIncurredButCovered()")
+            )
+        );
+        vm.label(mapleLoan, "mapleLoan");
+
+        // Pool delegate funds the loan
+        // The pool has enough cash to fund the loan because the PCVDeposit
+        // did a deposit() in the setUp() step.
+        vm.prank(MainnetAddresses.MPL_ORTHOGONAL_POOL_DELEGATE);
+        IMaplePool(maplePool).fundLoan(
+            mapleLoan,
+            mapleDebtLockerFactory,
+            targetUsdcBalance
+        );
+
+        // Borrower pulls loan amount
+        assertEq(usdc.balanceOf(address(this)), 0);
+        IMapleLoan(mapleLoan).drawdownFunds(targetUsdcBalance, address(this));
+        assertEq(usdc.balanceOf(address(this)), targetUsdcBalance);
+
+        assertEq(
+            IMapleLoan(mapleLoan).nextPaymentDueDate(),
+            block.timestamp + paymentInterval
+        );
+        assertEq(IMapleLoan(mapleLoan).gracePeriod(), gracePeriod);
+
+        // Pool Manager triggers a loan default
+        vm.warp(block.timestamp + gracePeriod + paymentInterval + 1000);
+        vm.prank(MainnetAddresses.MPL_ORTHOGONAL_POOL_DELEGATE);
+        IMaplePool(maplePool).triggerDefault(mapleLoan, mapleDebtLockerFactory);
+
+        // Pool Manager claims to update accounting
+        vm.prank(MainnetAddresses.MPL_ORTHOGONAL_POOL_DELEGATE);
+        vm.expectEmit(true, false, false, false, maplePool);
+        emit DefaultSuffered(mapleLoan, 123, 456, 789, 101);
+        IMaplePool(maplePool).claim(mapleLoan, mapleDebtLockerFactory);
+
+        // First-loss capital absorbed the default
+        assertEq(usdcDeposit.balance(), targetUsdcBalance);
+
```

Comment:
we should add an additional assertion here that the pcv controller can deposit again here even after losses are sustained.